build/tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
 // +build tools
```

Comment:
Looks like the build tag style is slightly different.
Will probably need to fix these up individually...

Reply:
Possibly, but not in our current invocation of it: https://github.com/kubernetes/kubernetes/blob/2584a7c0bb27a5029c3ac24b496a20da58edca35/hack/update-gofmt.sh#L46

`//go:build tools` seems like it should be equivalent to `// +build tools`, but the new tag prepended instead of replaced.

Reply:
> //go:build tools seems like it should be equivalent to // +build tools

I'm confused… according to https://tip.golang.org/doc/go1.17#gofmt and https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md they are equivalent, right?

Reply:
prepending is the expected behavior… the old style is maintained for compatibility with older go versions

Reply:
Okay, perfect (re: prepend vs replace).
(I'm AFK, so didn't have the relnotes in front of me.)